osbs/build/build_requestv2.py

```diff
@@ -644,6 +644,10 @@ def __init__(self, build_json_store, outer_template=None):
         self.user_params = SourceContainerUserParams(self.build_json_store)
 
     def render(self, validate=True):
+        koji_task_id = self.user_params.koji_task_id.value
```

Comment:
Can you move this block to superclass and update `BuildRequestV2.render` accordingly?  This is shared functionality between regular image builds and source container image builds, so it should be in common class.

Reply:
moved to common